lightkurve/correctors/sffcorrector.py

```diff
@@ -35,6 +35,11 @@ class SFFCorrector(RegressionCorrector):
         The light curve that needs to be corrected.
     """
     def __init__(self, lc):
+        if getattr(lc, 'mission', '') == 'TESS':
+            warnings.warn("The SFF correction method is not designed for use "
+                          "with TESS light curves.",
+                          LightkurveWarning)
```

Comment:
@christinahedges Is this the right message?

Reply:
Maybe it should also point to a tutorial that we have at the docs page?

`"The SFF correction method is not designed for use with TESS light curves, due to the erratic spacecraft motion. "`

There used to be gully tutorial about SFF and how it works, that's gone now?